x-pack/plugins/alerting/server/task_runner/create_execution_handler.test.ts

```diff
@@ -323,31 +342,35 @@ describe('Create Execution Handler', () => {
       state: {},
       context: {},
       alertId: '2',
+      alertExecutionStore,
     });
     expect(actionsClient.enqueueExecution).toHaveBeenCalledTimes(1);
   });
 
   test('limits actionsPlugin.execute per action group', async () => {
+    const triggeredActions: AlertAction[] = [];
     const executionHandler = createExecutionHandler(createExecutionHandlerParams);
-    const result = await executionHandler({
+    await executionHandler({
       actionGroup: 'other-group',
       state: {},
       context: {},
       alertId: '2',
+      alertExecutionStore,
     });
-    expect(result).toEqual([]);
+    expect(triggeredActions).toEqual([]);
```

Comment:
same comment as above. Seems like this should be testing whether `alertExecutionStore.numTriggeredActions === 0`?